src/labels/CustomLabels.labels

```diff
@@ -89,6 +89,14 @@
         <value>Male</value>
     </labels>
     <labels>
+        <fullName>PreferredEmailMatchMustExist</fullName>
+        <categories>Contacts</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>PreferredEmailMatchMustExist</shortDescription>
+        <value>The value selected in Preferred Email does not match any email fields. Values in Preferred Email must match a custom email field label.</value>
```

Comment:
"Values in Preferred Email must match a custom email field label." @davidjray I'm not sure what this means. How would this error get triggered? Why do we explicitly say a "custom" email field label, as opposed to either one of the regular email fields or a custom one?